build/edge/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BUILD_FROM=golang:1.12-alpine3.9
-ARG RUN_FROM=docker
+ARG RUN_FROM=alpine3.9
```

Comment:
@zhanghongtong please keep it `docker` here.
It will be used in the k8s deployment.
Could you modify somewhere in the dockercompose files?
Thanks.

Reply:
But I see that [04-deployment-edgenode.yaml already contains a docker:dind image](https://github.com/kubeedge/kubeedge/blob/92d87401fc6c97526e34c2f5ea8c7e2418aeb719/build/edge/kubernetes/04-deployment-edgenode.yaml#L44). Does k8s deployment require two docker containers?

Reply:
No, actually the edge node container is a docker client. 
and the docker:dind is the docker daemon server.

Reply:
+1 we require docker-in-docker feature.

Reply:
Nice catch @edisonxiang

Reply:
In my experiment, only need to set `edged.docker-address: tcp://dind:2375` in edgecore to successfully create the docker client, so I don't think we need two docker containers.
```
$ sudo docker exec edge_dind_1 sh -c "docker version"
Client: Docker Engine - Community
 Version:           18.09.6
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        481bc77
 Built:             Sat May  4 02:33:34 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.6
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       481bc77
  Built:            Sat May  4 02:41:08 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

Reply:
Docker-in-docker will run in the edge_dind_1 container
```
$ sudo docker exec edge_dind_1 sh -c "docker ps -a"
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
647788a5f308        nginx:1.15.12       "nginx -g 'daemon of…"   2 hours ago         Up 2 hours                              k8s_nginx_nginx-deployment-869d4ffccf-nlc6q_default_5a53a843-754f-11e9-9640-027fa4c0ef2a_0
```

Reply:
yep. I mean we need `docker client` also called `cli` in the edgecore image.

Reply:
we will not run another container in the edgecore container:)

Reply:
Oh, I know, let me modify it

Reply:
The docker container is not in the edgecore container. It is a cluster with edgecore.

Reply:
@zhanghongtong yes:)

Reply:
I have modified it, you can check it again.